.github/workflows/bin/templates/cron-workflow-template.j2

```diff
@@ -0,0 +1,168 @@
+        run: |
+          source "$CONDA/etc/profile.d/conda.sh"
+          conda activate base
+          wget ${{ matrix.pkg.url }}
+          sudo conda build -q -c $Q2_CHANNEL -c conda-forge -c bioconda -c defaults --extra-deps ${{ needs.install_env.outputs.metapackage_spec }} -t "*.tar.bz2"
+  phone_home:
+    needs: test_package
+    runs-on: {{{ input_os }}}-latest
+    steps:
+      - name: checkout source
+        uses: actions/checkout@v2
+
+      - name: configure conda and friends
+        run: |
+          source "$CONDA/etc/profile.d/conda.sh"
+          conda activate base
+          conda config --set always_yes yes
+          conda config --set changeps1 no
+          conda config --set quiet true
+          sudo conda upgrade -n base -q -y -c defaults --override-channels conda
+          sudo conda install -n base -q -y -c defaults --override-channels conda-build yq
+          conda info -a
+          conda config --show
+      - name: phone home
+        env:
+          LIBRARY_TOKEN: ${{ secrets.LIBRARY_TOKEN }}
+          DATA_FILE:  ${{ env.epoch }}/staged/${{ env.distro }}/data.yaml
+          BUILD_FILE: ${{ env.epoch }}/staged/${{ env.distro }}/conda_build_config.yaml
+          META_FILE: ${{ env.epoch }}/staged/${{ env.distro }}/meta.yaml.jinja
+        run: |
+          source "$CONDA/etc/profile.d/conda.sh"
+          conda activate base
+          phone_home=$(yq ".meta.phone_home" $DATA_FILE)
+          if [ "$phone_home" = "false" ]
+          then
+              echo "skipping library upload due to disabled phone_home"
+              exit 0
+          fi
+          metapackage_version=$(yq -r .version $DATA_FILE)
+          package_versions=$(jq -cs '.[1] as $build | .[0] as $data | .[2] as $meta | [$data.run[] | {(.): $build[. | split("-") | join("_")][0]}] | add + {($meta.name):$data.version}' \
+                              <(yq -cj '.' $DATA_FILE) \
+                              <(yq -cj '.' $BUILD_FILE) \
+                              <(head -n 2 $META_FILE | yq -cj '.package'))
+          resp=$(curl \
+            --silent \
+            --include \
+            --data-urlencode "token=$LIBRARY_TOKEN" \
+            --data-urlencode "version=$metapackage_version" \
+            --data-urlencode "package_versions=$package_versions" \
+            --data "run_id=$GITHUB_RUN_ID" \
+            --data "distro=${{ env.distro }}" \
+            --data "epoch=${{ env.epoch }}" \
+            --data "artifact_name=${{ env.distro }}-{{{ op_sys }}}" \
+            --header "Content-Type: application/x-www-form-urlencoded" \
+            --request POST \
+            https://library.qiime2.org/api/v1/packages/pass/
+          )
+          code=$(echo $resp | grep HTTP | awk '{print $2}')
+          if [[ $code -ne 200 ]]
+          then
+            echo "whoops"
+            echo $resp
+            exit 1
+          fi
```

Comment:
Please incorporate the latest changes to the cron workflow (https://github.com/qiime2/package-integration/commit/061dd631c87aa5d608fee04b9d86ad6de109fc98).